lib/src/sdk/sdk_service.dart

```diff
@@ -0,0 +1,32 @@
+// ignore_for_file: public_member_api_docs, sort_constructors_first
```

Comment:
Why is this file called SDK service? Also let's format the files properly with proper return statements and spacing.